services/powershellgallery/powershellgallery.service.js

```diff
@@ -135,4 +140,78 @@ class PowershellGalleryDownloads extends BaseXmlService {
   }
 }
 
-module.exports = { PowershellGalleryVersion, PowershellGalleryDownloads }
+class PowershellGalleryPlatformSupport extends BaseXmlService {
+  static get category() {
+    return 'platform-support'
+  }
+
+  static get route() {
+    return {
+      base: 'powershellgallery/ps',
+      pattern: ':packageName',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'PowerShell Gallery',
+        namedParams: { packageName: 'Az.Storage' },
+        staticExample: this.render({
+          platforms: ['windows', 'macos', 'linux'],
+        }),
+      },
+    ]
+  }
+
+  static render({ platforms }) {
+    return {
+      label: 'platform',
```

Comment:
It's better to put `label` in `defaultBadgeData`. That way it'll still be shown in the error case.

```js
static get defaultBadgeData() {
  return {
    label: 'platform',
  }
}
```

Reply:
added